packages/ai-jsx/src/lib/openai.tsx

```diff
@@ -492,31 +539,44 @@ export async function* OpenAIChatModel(
 
     // TS doesn't realize that the Stream closure can make `delta` be `null`.
     // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-    if (delta?.function_call) {
+    while (delta?.tool_calls) {
```

Comment:
Q: so this was a breaking change or does this name change only happen at their SDK layer (not the API layer)?

Reply:
I think if you use `functions` on the request, they use `function_call` in the response; if you use `tools` on the request, they use `tool_ calls` in the response.